corehq/apps/hqwebapp/static/hqwebapp/js/bootstrap5/inactivity.js

```diff
@@ -122,6 +124,17 @@ hqDefine('hqwebapp/js/bootstrap5/inactivity', [
         var isModalOpen = function (element) {
             // https://stackoverflow.com/questions/19506672/how-to-check-if-bootstrap-modal-is-open-so-i-can-use-jquery-validate
             return (element.data('bs.modal') || {}).isShown;
+
+        const showModal = function ($element) {
+            if ($element.length) {
+                bootstrap.Modal.getOrCreateInstance($element).show();
+            }
+        };
+
+        const hideModal = function ($element) {
+            if ($element.length) {
+                bootstrap.Modal.getOrCreateInstance($element).hide();
```

Comment:
Yeah, I was just doing line-for-line replacements, but this does read funny. I added da5fe58fa6b to keep track of the bootstrap modals, rather than getOrCreate-ing them every time they're needed.